src/operations/functions/shared.ts

```diff
@@ -28,4 +28,9 @@ export interface FunctionOptions {
   onNull?: boolean;
 
   parallel?: 'UNSAFE' | 'RESTRICTED' | 'SAFE';
+
+  set?: Array<{
+    name: string;
+    value: LiteralUnion<'FROM CURRENT'>;
+  }>;
```

Comment:
**question (if-minor):** I read a bit through the docs
might it be better to e.g. use something like this?

```ts
  set?: Array<{
    value: 'FROM CURRENT';
  } | {
    name: string; // should name be called configurationParameter? https://www.postgresql.org/docs/current/sql-createfunction.html
    value: string;
  }>;
```

Reply:
Or even simpler: 
```
  set?: Array<'FROM CURRENT' | {
    name: string; // should name be called configurationParameter? https://www.postgresql.org/docs/current/sql-createfunction.html
    value: string;
  }>;
```

Reply:
The `name` parameter has to stay here, as the `configuration_parameter` to override is specified there. But we can rename it to `configurationParameter`.